bridge_ui/src/components/Stats/TVLStats.tsx

```diff
@@ -127,26 +127,30 @@ const TVLStats = () => {
     return chainIds;
   }, [cumulativeTVL]);
 
-  const handleDisplayByChange = useCallback((event, nextValue) => {
+  const handleDisplayByChange = useCallback((event: React.MouseEvent<HTMLElement>, nextValue: string) => {
     if (nextValue) {
       setDisplayBy(nextValue);
     }
   }, []);
 
   const handleTimeFrameChange = useCallback(
-    (event) => setTimeFrame(event.target.value),
+    (event: React.ChangeEvent<HTMLInputElement>) => setTimeFrame(event.target.value),
```

Comment:
We use `React.ChangeEvent` here, and `ChangeEvent` below, line 142. Let's use the latter everywhere?

Reply:
Yes, unmodified copied code. Fixed.